packages/atst/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@eth-optimism/atst",
+  "version": "0.0.0",
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "src/index.ts",
+  "module": "dist/index.cjs",
+  "license": "MIT",
+  "bin": {
+    "atst": "./dist/cli.js"
+  },
+  "scripts": {
+    "cli:read": "node ./dist/cli.js read",
+    "clean": "rm -rf ./node_modules && rm -rf ./dist && rm -rf ./coverage",
+    "build": "yarn tsup",
+    "lint": "yarn lint:fix && yarn lint:check",
+    "lint:check": "eslint . --max-warnings=0",
+    "lint:fix": "yarn lint:check --fix",
+    "test": "vitest",
+    "typecheck": "yarn tsc --noEmit"
+  },
+  "dependencies": {
+    "@eth-optimism/contracts-periphery": "^1.0.7",
+    "@wagmi/core": "^0.9.2",
+    "cac": "^6.7.14",
```

Comment:
this is a command line library.  I used it because wagmi used it in their cli so I am familiar with it.  I might swap it out for something different if I see something better out there